app/javascript/packages/analytics/.gitignore

```diff
@@ -0,0 +1 @@
+digital-analytics-program.js
```

Comment:
should we add this to an `eslint` ignore too?

Reply:
> should we add this to an `eslint` ignore too?

Yeah I think so 👍

Reply:
> should we add this to an `eslint` ignore too?

Added to `.eslintignore` in 4c4fb997ea.